R/classes.R

```diff
@@ -574,11 +574,13 @@ nexml_namespaces <-
     "cdao"  = "http://purl.obolibrary.org/obo/cdao.owl",
     "xsd"   = "http://www.w3.org/2001/XMLSchema#",
     "dc"    = "http://purl.org/dc/elements/1.1/",
-  "dcterms" = "http://purl.org/dc/terms/",
+    "dcterms" = "http://purl.org/dc/terms/",
+    "ter" = "http://purl.org/dc/terms/",
     "prism" = "http://prismstandard.org/namespaces/1.2/basic/",
     "cc"    = "http://creativecommons.org/ns#",
     "ncbi"  = "http://www.ncbi.nlm.nih.gov/taxonomy#",
-    "tc"    = "http://rs.tdwg.org/ontology/voc/TaxonConcept#")
+    "tc"    = "http://rs.tdwg.org/ontology/voc/TaxonConcept#",
+    "ns"    = "http://www.nexml.org/2009")
```

Comment:
@hlapp right, I think things should work without that change.  in principle any of these could be different of course; these are really just here as the namespace prefixs that RNeXML uses when generating common metadata (e.g. so a user can declare a ''title' without having to fret about what namespace the word 'title' belongs to).